src/dkg/complaint.rs

```diff
@@ -47,7 +48,7 @@ impl<C: CipherSuite> Complaint<C> {
         let a1 = C::G::generator().mul(r);
         let a2 = accused_pk.mul(r);
 
-        let hasher = <DefaultFieldHasher<Sha256, 128> as HashToField<Scalar<C>>>::new(
+        let hasher = <DefaultFieldHasher<Sha256, HASH_SEC_PARAM> as HashToField<Scalar<C>>>::new(
```

Comment:
Note that this is "technically" unrelated to these PR described changes, though harmless as `HASH_SEC_PARAM` was originally aimed at being used at every relevant places, including here.